### PR TITLE
[4.0] Hide language column if not multilanguage in modal

### PR DESCRIPTION
**Pull request by Quy**
### Summary of Changes
Check if multilanguage is enabled to display the language column.

### Testing Instructions
Create a menu item of Single Article.
Click Select button to display modal.

Create a menu item of Menu Item Alias.
Click Select button to display modal.

Create a contact.
Create a menu item of Single Contact.
Click Select button to display modal.

Create a feed.
Create a menu item of Single News Feed.
Click Select button to display modal.

### Actual result BEFORE applying this Pull Request
Language column is displayed.

### Expected result AFTER applying this Pull Request
No language column.

### CODE REVIEW

**Comment by ceford** [comment]
Is there a way to toggle a multi-lingual site to/from single language? I guess I should be keeping a separate mono-lingual site to run tests like this.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30344">issues.joomla.org/tracker/joomla-cms/30344</a>.</sub>

File: administrator/components/com_content/tmpl/articles/modal.php [hunk 1]
```diff
@@ -74,9 +74,11 @@
 						<th scope="col" class="w-10 d-none d-md-table-cell">
 							<?php echo HTMLHelper::_('searchtools.sort', 'JGRID_HEADING_ACCESS', 'a.access', $listDirn, $listOrder); ?>
 						</th>
-						<th scope="col" class="w-15">
-							<?php echo HTMLHelper::_('searchtools.sort', 'JGRID_HEADING_LANGUAGE', 'language', $listDirn, $listOrder); ?>
-						</th>
+						<?php if (Multilanguage::isEnabled()) : ?>
```
Comment (alikon) [inline administrator/components/com_content/tmpl/articles/modal.php]: May i ask you to run `Multilanguage::isEnabled()` only once

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 1c0cf6305e3f5e664ae4242feff197239d3aac34<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30344">issues.joomla.org/tracker/joomla-cms/30344</a>.</sub>

**Comment by infograf768** [comment]
@Quy 
I guess should be done the same for all modals like this one. Contacts, Menus (items), Newsfeeds

File: administrator/components/com_content/tmpl/articles/modal.php (hunk 1 above)
Comment (infograf768) [reply administrator/components/com_content/tmpl/articles/modal.php]: We need this otherwise we get
Screen Shot 2020-08-12 at 10 04 33

Comment (alikon) [reply administrator/components/com_content/tmpl/articles/modal.php]: i mean 
https://github.com/joomla/joomla-cms/pull/30344/files#diff-a42b2b0c3999aaf0517b60ea5d19f8ccR77
https://github.com/joomla/joomla-cms/pull/30344/files#diff-a42b2b0c3999aaf0517b60ea5d19f8ccR148

Comment (infograf768) [reply administrator/components/com_content/tmpl/articles/modal.php]: we need both, otherwise, as my screenshot shows, we get an extraneous `<td>`

Comment (Quy) [reply administrator/components/com_content/tmpl/articles/modal.php]: Thank you both!

Comment (Quy) [reply administrator/components/com_content/tmpl/articles/modal.php]: Please advise where I can hide the Language dropdown in Filter Options. Thanks.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 881dd1186b13c4f6ca39f07802bea212f7329e0b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30344">issues.joomla.org/tracker/joomla-cms/30344</a>.</sub>

**Comment by richard67** [comment]
@Quy Any reason why it is in daft status? We have 2 good tests, not sure if I shall set RTC or wait for something.

**Comment by richard67** [comment]
Ah, no only 1 test. @infograf768 could you test again ?

**Comment by Quy** [comment]
Menus (items) is a special case. I need to investigate further to see if it needs to be done similar to default.php in the same directory. Please hold off testing. Thanks.

**Comment by Quy** [comment]
False alarm. Brian's test is good. One more test needed.

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 9e59744e6059f4893df7404a9a71e0e95f34a760<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30344">issues.joomla.org/tracker/joomla-cms/30344</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30344">issues.joomla.org/tracker/joomla-cms/30344</a>.</sub>

**Comment by brianteeman** [comment]
@quy its best practice not to merge your own PR.

**Pull request merged**